### ISSUE TITLE
'Reformat Code' disobeys the Tidyverse Style Guide

### ISSUE THREAD

**Issue by wurli** [rstudio/rstudio#12661]
From the [Tidyverse Style Guide](https://style.tidyverse.org/syntax.html?q=long%20lines#long-lines):

> If a function call is too long to fit on a single line, use one line each **for the function name**, each argument, and the closing `)`. This makes the code easier to read and to change later.

I'm specifically talking about putting the function name on its own line. In my opinion, this makes long scripts _much_ easier to read. E.g:

##### Bad:
```
iris |>
  summarise(Sepal.Area = Sepal.Width * Sepal.Length,
            Petal.Area = Petal.Width * Petal.Length)
```

##### Good:
```
iris |>
  summarise(
    Sepal.Area = Sepal.Width * Sepal.Length,
    Petal.Area = Petal.Width * Petal.Length
  )
```

Unfortunately the 'Reformat Code' helper favours the style of the first example. If this could be changed I would be eternally grateful.

Thanks!

**Comment by ronblum** [comment]
@wurli Thank you for raising this! And in this case, reformatting the "good" code transforms it into the "bad" version. In fact, this happens with the example used in Style Guide. I'll mark this for review as continue to develop RStudio.

**Comment by wurli** [comment]
Very glad to hear that! Thank you very much 😃

**Comment by olivroy** [comment]
Hi, in case you don't know about it, there is an addin that does that much better. Just install the styler package
styler.r-lib.org

**Comment by wurli** [comment]
Thanks, I'm aware of the addin and {styler} is what I tend to use. The reason I'm keen to get this issue fixed is because it's slightly easier to direct other colleagues to use the built-in version, and it's also much easier to embed good styling practices if you don't need to tell people to avoid the most obvious tools.

**Comment by olivroy** [comment]
I agree, but as far I can remember, the native tool was never really good.
And with the amount of work it took to make styler viable and functioning, I doubt the native tool will get as good as styler.

**Comment by wurli** [comment]
Thinking about this some more, how feasible would it be for 'Reformat Code' to simply call a function from styler under the hood? Obviously this would have to deal with users potentially not having styler installed. 

Is there precedent for this? Memory is hazy, but RStudio does seem to have some level of baked-in integration with certain packages (knitr, shiny, DBI to name a few). Why not styler too?

**Comment by kevinushey** [comment]
Yeah, at this point we ought to just use `styler` for formatting.

**Comment by wurli** [comment]
I think with the release of Air we can consider this closed :)
